R/Reactant/build_tarballs.jl

```diff
@@ -450,7 +457,8 @@ for (i,build) in enumerate(builds)
     build_tarballs(i == lastindex(builds) ? non_platform_ARGS : non_reg_ARGS,
                    name, version, build.sources, build.script,
                    build.platforms, build.products, build.dependencies;
-                   preferred_gcc_version=v"10", preferred_llvm_version=v"18", julia_compat="1.6",
-                   augment_platform_block, lazy_artifacts=true)
+                   preferred_gcc_version=build.preferred_gcc_version, preferred_llvm_version=v"18", julia_compat="1.10",
+                   # We use GCC 13, so we can't dlopen the library during audit
```

Comment:
I just realised I meant to write "because we can't..." but let's not fix this now, it'll take forever to finish the build otherwise 🥲